lib/vanagon/platform/windows.rb

```diff
@@ -66,6 +66,38 @@ def generate_nuget_package(project)
         "#{@copy} $(tempdir)/#{project.name}/#{project.name}-#{@architecture}.#{nuget_package_version(project.version, project.release)}.nupkg ./output/#{target_dir}/#{nuget_package_name(project)}"]
       end
 
+      # The specific bits used to generate a windows msi package for a given project
+      # Have changed this to reflect the overall commands we need to generate the package.
+      # Question - should we break this down into some simpler Make tasks ?
+      # 1. Heat the directory tree to produce the file list
+      # 2. Compile (candle) all the wxs files into wixobj files
+      # 3. Run light to produce the final MSI
+      #
+      # @param project [Vanagon::Project] project to build a msi package of
+      # @return [Array] list of commands required to build an msi package for the given project from a tarball
+      def generate_msi_package(project)
+        target_dir = project.repo ? output_dir(project.repo) : output_dir
+        cg_name = "compfiles"
+        dir_ref = "INSTALLDIR"
```

Comment:
Oh wait, yeah, I was. According to https://github.com/puppetlabs/puppet_for_the_win/blob/b515d41c89ade27d8b68d02ccbf24cf7ebed5a71/tasks/windows/windows.rake#L102, this makes sense